README.md

```diff
@@ -135,6 +136,12 @@ The [fluent-mixin-config-placeholders](https://github.com/tagomoris/fluent-mixin
 
 Format of the time used in the file name. Default is '%Y%m%d'. Use '%Y%m%d%H' to split files hourly.
 
+### `calculate_checksums`
+
+Whether to calculate MD5 checksum and provide it in a header for the blob service.
+
+Pro tip: Turn off checksum calculation for FIPS enabled environment.
```

Comment:
Maybe rephrase that to `you prob want to turn this off in FIPS enabled environments`?